job_spec/INSAR_ISCE.yml

```
@@ -37,6 +37,10 @@ INSAR_ISCE:
           - HRRR
           - HRES
           - GMAO
+          - ERA5
+          - NCMR
+          - ERA5
+          - ERA5T
```

```suggestion
          - ERA5
          - ERA5T
          - GMAO
          - HRES
          - HRRR
          - NCMR
```

I think ERA5 is duplicated here. Also, might be nice to alphabetize these.